cordova/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hk.goodcity.adminstaging",
+  "version": "0.14.017",
+  "displayName": "S. Admin GoodCity",
+  "cordova": {
+    "platforms": [
+      "ios",
+      "android",
+      "windows"
+    ]
+  },
+  "dependencies": {
+    "android-versions": "^1.3.0",
+    "cordova-android": "~5.1.1",
```

Comment:
Should this be `6.3.0` or `5.1.1`?

Reply:
@abulsayyad123 , why do we need this file?? I don't see this file in any other repo.